keps/sig-apps/4368-support-managed-by-for-batch-jobs/README.md

```diff
@@ -140,7 +140,7 @@ controller, and delegate the status synchronization to the Kueue controller.
 ### Non-Goals
```

Comment:
Earlier in the doc, make sure to check all the boxes in the `## Release Signoff Checklist`